en/debug/consoles.md

```diff
@@ -45,6 +47,40 @@ nsh> free
 nsh> dmesg
 ```
 
+Below are a couple of commands which can be used in the [NuttShell](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=139629410) to get insights of the system.
+
+This NSH command provides the remaining free memory:
+
+```bash
+free
```

Comment:
in future, might be worth turning each of these into headings, showing the output, and explaining how you interpret the result (i.e. a real world debugging case to show their usefulness)